src/app.rs

```diff
@@ -172,6 +176,12 @@ impl Application {
                 }
                 config.healthchecks.set_require_healthy(require_healthy);
 
+                #[cfg(feature = "datadog")]
+                // Augment config to enable observability within Datadog, if applicable.
+                if config::datadog::try_attach(&mut config).is_ok() {
+                    info!(DATADOG_OBSERVABILITY_MESSAGE);
+                }
```

Comment:
It appears every call to `try_attach` is followed by this message. This suggests either moving the condition into a trivial wrapper function or right into `try_attach`.

Reply:
Thanks! I originally had it that way, and then figured we might want different messages for reloading versus first loading. I didn't wind up doing trying to figure out a first load vs. a reload, so this has been moved back into the function per your suggestion.